hyperframe/frame.py

```diff
@@ -343,17 +343,25 @@ class SettingsFrame(Frame):
     # We need to define the known settings, they may as well be class
     # attributes.
     #: The byte that signals the SETTINGS_HEADER_TABLE_SIZE setting.
-    HEADER_TABLE_SIZE             = 0x01
+    HEADER_TABLE_SIZE      = 0x01
     #: The byte that signals the SETTINGS_ENABLE_PUSH setting.
-    ENABLE_PUSH                   = 0x02
+    ENABLE_PUSH            = 0x02
     #: The byte that signals the SETTINGS_MAX_CONCURRENT_STREAMS setting.
-    MAX_CONCURRENT_STREAMS        = 0x03
+    MAX_CONCURRENT_STREAMS = 0x03
     #: The byte that signals the SETTINGS_INITIAL_WINDOW_SIZE setting.
-    INITIAL_WINDOW_SIZE           = 0x04
+    INITIAL_WINDOW_SIZE    = 0x04
     #: The byte that signals the SETTINGS_MAX_FRAME_SIZE setting.
-    SETTINGS_MAX_FRAME_SIZE       = 0x05
+    #: .. deprecated:: 3.2.0
+    #:    Use :data:`MAX_FRAME_SIZE <SettingsFrame.MAX_FRAME_SIZE>` instead.
+    MAX_FRAME_SIZE         = 0x05
     #: The byte that signals the SETTINGS_MAX_HEADER_LIST_SIZE setting.
-    SETTINGS_MAX_HEADER_LIST_SIZE = 0x06
+    #: .. deprecated:: 3.2.0
+    #     Use :data:`MAX_HEADER_LIST_SIZE <SettingsFrame.MAX_HEADER_LIST_SIZE>` instead.
```

Comment:
I think these comments are in the wrong place. We want the *old* comment to be on the *new* value, and the *old* values to have the "deprecated" comment above them. Sorry I wasn't clearer!